backend/routes/post.py

```
@@ -18,7 +18,10 @@
 from core.connection_crud import get_connections
 from crud.notification import create_notification
 from AI.moderation import moderate_text
-from services.services import get_newer_posts, get_user_like_status, get_comments_for_post, get_post_additional_data, create_post_entry, update_post_content, remove_old_file_if_exists, save_upload_file, generate_secure_filename, validate_file_extension, extract_hashtags, send_post_notifications, get_post_by_id, get_post_and_event, update_post_and_event, try_convert_datetime, format_updated_event_response
+from services.services import get_newer_posts, get_user_like_status, get_comments_for_post, get_post_additional_data, create_post_entry, update_post_content, get_post_by_id
```

Ruff (0.8.2)

`services.services.get_comments_for_post` imported but unused

Remove unused import: `services.services.get_comments_for_post`

(F401)